sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.actions.headers;
+
+import com.example.wiring.actions.echo.Message;
+import kalix.javasdk.action.Action;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * Action with the same name in a different package.
```

Comment:
Could we have a test that verifies that the service has two Actions sharing the same name? I don't know how hard it will be to verify it, but it will be nice if we make it waterproof.

Reply:
This class in some sense testing this already. We have the same class in the `com.example.wiring.actions.echo` package so without the fix, all integrations tests will fail, because of proxy exception. Is that enough?

Reply:
If you remove this class tests will still pass. 

What I mean by waterproofing is to have a check that indeed the final descriptor do have two services with the same name.

But that might be hard to achieve.

Reply:
ok, I get it, let me check what I can do

Reply:
After a few attempts - I give up, maybe I will figure this out later. I only added one assertion for the filedescriptor name.